networkx/algorithms/centrality/current_flow_closeness.py

```diff
@@ -59,22 +74,30 @@ def current_flow_closeness_centrality(G, weight=None, dtype=float, solver="lu"):
     .. [1] Ulrik Brandes and Daniel Fleischer,
        Centrality Measures Based on Current Flow.
        Proc. 22nd Symp. Theoretical Aspects of Computer Science (STACS '05).
-       LNCS 3404, pp. 533-544. Springer-Verlag, 2005.
+       LNCS 3404, pp. 533--544. Springer-Verlag, 2005.
        https://doi.org/10.1007/978-3-540-31856-9_44
 
     .. [2] Karen Stephenson and Marvin Zelen:
        Rethinking centrality: Methods and examples.
-       Social Networks 11(1):1-37, 1989.
+       Social Networks 11(1):1--37, 1989.
        https://doi.org/10.1016/0378-8733(89)90016-6
     """
+    N = G.number_of_nodes()
+    if N == 0:
+        raise nx.NetworkXPointlessConcept("centrality is undefined for the null graph")
+    if N < 3:
+        raise nx.NetworkXError(
+            f"graph with {N} node{'' if N == 1 else 's'} has fewer than three nodes"
+        )
+
     if not nx.is_connected(G):
-        raise nx.NetworkXError("Graph not connected.")
+        raise nx.NetworkXError("graph is not connected")
```

Comment:
```suggestion
        raise nx.NetworkXError("Graph is not connected")
```

Reply:
This feels like a nit, but is there a style guide for exceptions in NetworkX? I was following the recommendations from the Python base library, i.e. no punctuation, no upper case, but if the standard here is different I'm not going to die on that hill.

For reference:
```python
>>> int("not an int")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: 'not an int'
```